### ISSUE TITLE
[Suggestion] Configurable number of slots for the Food Containers

### ISSUE THREAD

**Issue by Xiaminou** [squeek502/SpiceOfLife#140]
When I include Spice of Life in my modpack I usually also change its configuration to be the last 6 foods you ate instead of the default number. Why? Because I want so that you still only have 1 inventory slot occupied by food, and the Lunch Box has 6 slots.
Now I always play it alongside mods like Pam's Harvestcraft, Cooking for blockheads, Farming for blockheads which give you a lot of choices and possibilities when it comes to foods.
However because I only want it to occupy one slot of my inventory I am still limited to 6 foods history.

TL;DR: How about adding a config option for the number of slots in the Lunch Box and the Lunch Bag?

**Comment by pauloelr** [comment]
Please make it

**Comment by pauloelr** [comment]
Well, why not do it myself?

Sent a PR with an attempt to implement it
